### PR TITLE
chore(test): enhancing trace naming with workerIndex to distinguish between files with the same name

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
Enhances the naming of the trace/video files with the worker index in order to be able to distinguish between trace files from the same test file but from different workers.

### CODE REVIEW

File: tests/playwright/src/runner/podman-desktop-runner.ts [hunk 1]
```diff
@@ -400,9 +400,16 @@ export class Runner {
...
     this._videoAndTraceName = name;
 
-    if (test.info().retry > 0) {
+    if (test?.info()?.retry && test.info()?.retry > 0) {
       this._videoAndTraceName += `_retry-${test.info().retry}`;
+      return;
     }
+
+    this._videoAndTraceName += `_w${test?.info()?.workerIndex}`;
+  }
```
[the remark this answers is not in the document]
Comment (cbr7) [reply tests/playwright/src/runner/podman-desktop-runner.ts:409]: Not really, because the retry counter in the naming makes it distinguishable, you would get retry-1, retry-2 etc, so retry would do the job that _w0 or _w1 would be doing for test runs that are not retries

[the remark this answers is not in the document]
Comment (cbr7) [reply tests/playwright/src/runner/podman-desktop-runner.ts:409]: It's not even a narrow edge case here since the test framework operates with only one worker, since the podman desktop application cannot have multiple instances running at the same time, so parallel test execution doesn't exist here.

**Review by serbangeorge-m — APPROVED** [review]
